config.go

```diff
@@ -383,10 +383,21 @@ func GetDNSConfig() (*tailcfg.DNSConfig, string) {
 		if viper.IsSet("dns_config.nameservers") {
 			nameserversStr := viper.GetStringSlice("dns_config.nameservers")
 
-			nameservers := make([]netip.Addr, len(nameserversStr))
-			resolvers := make([]*dnstype.Resolver, len(nameserversStr))
+			nameservers := []netip.Addr{}
+			resolvers := []*dnstype.Resolver{}
+
+			for _, nameserverStr := range nameserversStr {
+				// Search for a nextdns pattern
+				if strings.HasPrefix(nameserverStr, "nextdns:") {
+					nextdnsID := strings.TrimPrefix(nameserverStr, "nextdns:")
+					resolvers = append(resolvers, &dnstype.Resolver{
+						Addr: fmt.Sprintf("https://dns.nextdns.io/%s", nextdnsID),
+					})
+
+					continue
+				}
```

Comment:
This parsing logic might be a bit naive. Would you want this to be improved and maybe even use regex?

Reply:
For context, the idea here is to provide this to the list of nameservers like this:

```yaml
dns_config:
  nameservers:
    - nextdns:abcdef # where abcdef is the NextDNS ID
```